sdk/python/feast/infra/offline_stores/offline_utils.py

```diff
@@ -217,3 +220,29 @@ def get_offline_store_from_config(offline_store_config: Any) -> OfflineStore:
     class_name = qualified_name.replace("Config", "")
     offline_store_class = import_class(module_name, class_name, "OfflineStore")
     return offline_store_class()
+
+
+def get_pyarrow_schema(
```

Comment:
could you rename this to `get_feature_view_batch_source_pyarrow_schema` or something more specific.

Reply:
done